tests/unit/roommanager.spec.js

```diff
@@ -0,0 +1,129 @@
+const fs = require('fs');
+const path = require('path');
+const roommanager = require("../../roommanager")({ on() {} }, require("../../storage"));
+const InfoExtract = require("../../infoextract");
```

Comment:
Aside: This is imported as PascalCase whereas roommanager is all lowercase?

Reply:
It's consistent with how these modules are imported in other files